proto/gnmi_ext/gnmi_ext.proto

```diff
@@ -46,6 +46,9 @@ enum ExtensionID {
   // New extensions are to be defined within this enumeration - their definition
   // MUST link to a reference describing their implementation.
 
+  // Juniper Telemetry header
+  EID_JUNIPER_TELEMETRY_HEADER = 1;
```

Comment:
Can you please add a URL to the proto definition / implementation of this header as per the above comment?